### PR TITLE
added a hover effect to download button

### PR DESCRIPTION
**Pull request by the3plet**
Made a hover effect to download button.
Do merge this branch if you like the feature

### CODE REVIEW

File: assets/css/styles.css [hunk 1]
```diff
@@ -37,3 +37,20 @@
...
+  transform: perspective(1px) translateZ(0);
+  box-shadow: 0 0 1px rgba(0, 0, 0, 0);
+  -webkit-transition-duration: 0.3s;
+  transition-duration: 0.3s;
+  -webkit-transition-property: transform;
+  transition-property: transform;
+}
+.hvr-grow:hover, .hvr-grow:focus, .hvr-grow:active {
+  -webkit-transform: scale(1.1);
+  transform: scale(1.1);
```
Comment (YoshithaRathnayake) [inline assets/css/styles.css:56]: Add a new line at the bottom

**Review by YoshithaRathnayake — CHANGES_REQUESTED** [review]

**Comment by YoshithaRathnayake** [comment]
Create a new issue for this @the3plet

**Comment by the3plet** [comment]
@YoshithaRathnayake  can you please remove spam label. 
Purposefully i have n't done any spamming commit in this pr. 
Can you please explain why you have marked it as spam

**Comment by YoshithaRathnayake** [comment]
I added the spam label bcs you didn't asked me to do this in an issue however I will remove spam label now bcs this is your first time no? @the3plet

**Pull request closed without merging**